karate-mock-servlet/src/test/java/mock/jersey/hello-http.feature

```diff
@@ -2,18 +2,17 @@ Feature: test that works un-changed for http as well as in-process servlet / moc
 
 Scenario: get hello
 
-When url demoBaseUrl
-And path 'hello'
-When method get
-Then status 200
-And match response == 'hello world'
+  When url demoBaseUrl
```

Comment:
this is a nit-pick - can you omit this file from the PR, it is just white-space

Reply:
Okay will do, getting my head around Git haha